products/oneplus.md

```diff
@@ -53,15 +54,15 @@ releases:
     releaseDate: 2023-02-07 #https://oxygenupdater.com/article/379/
     eoas: 2027-02-07 # approximation "4 major Android updates" https://community.oneplus.com/thread/1211291251581124608
     eol: 2028-02-07
-    discontinued: false
+    discontinued: true
     link: https://www.oneplus.in/11r
     supportedOxygenOSVersions: 13 - 14 # https://oxygenupdater.com/article/431/ https://community.oneplus.com/thread/1480591576202739713
 
 -   releaseCycle: "11"
     releaseDate: 2023-02-07 # https://oxygenupdater.com/article/379/
     eoas: 2027-02-07 # approximation "4 major Android updates" https://community.oneplus.com/thread/1211291251581124608
     eol: 2028-02-07
-    discontinued: false
+    discontinued: true
```

Comment:
```suggestion
    discontinued: false
```

Still listed on https://www.oneplus.com/us/oneplus-11 (even if out of stock), so should we really make this discontinued ?